modules/seqexec/server/src/main/scala/seqexec/server/gnirs/Gnirs.scala

```diff
@@ -180,4 +210,11 @@ object Gnirs {
     case _                      => None
   }
 
+  private def getFocus(config: Config): Either[ExtractFailure, GnirsController.Focus] =
+    config.extractInstAs[Focus](FOCUS_PROP).flatMap{ v =>
+      if (v.toString === FocusSuggestion.BEST_FOCUS.displayValue) GnirsController.Focus.Best.asRight
```

Comment:
Doesn't wart remover complain about `toString`?

Reply:
No. I am not happy with having to analyse the string value to figure out with type of focus setting to use, but I couldn't think of a better way.